### DISCUSSION TITLE
Customizable placeholder text

### DISCUSSION THREAD

**Discussion by asyncjason** [josdejong/svelte-jsoneditor#361] [Ideas]
Just wondering if it's possible to change the placeholder text in the tree mode?
I couldn't see anything in the docs. 
Would it be possible to have a placeholder in either mode?

**Comment by josdejong** [comment]
Thanks for your suggestion. I suppose you mean the text "Empty document" followed by 4 bullets of what you can do?

**Comment by asyncjason** [comment]
Yep that’s it

**Comment by josdejong** [comment]
Makes sense to be able to show your own "welcome text" there. Anyone interested in picking this up and think through what would be a nice API? It would be nice if you can specify not just plain text but HTML.

**Comment by asyncjason** [comment]
It would be a nice option if the placeholder text could also be an actual JSON document in tree mode, just greyed out...

**Comment by josdejong** [comment]
Moving this idea to the [Discussions > Ideas](https://github.com/josdejong/svelte-jsoneditor/discussions/categories/ideas) section since we are not actively working on it. Help implementing this would be welcome.
